src/client/labels-api.js

```diff
@@ -0,0 +1,103 @@
+const Label = require('../model/label');
+
+class LabelsApi {
+  /**
+   * @param {Object} options
+   */
+  constructor(opts) {
+    this.dryRun = opts.dryRun;
+    this.github = opts.github;
+    this.owner = opts.owner;
+    this.repo = opts.repo;
+
+    this.github.authenticate({
+      type: 'oauth',
+      token: opts.token,
+    });
+  }
+
+  /**
+   * @returns {Promise}
+   */
+  async getLabels() {
+    const labels = await this.github.paginate(
+      this.github.issues.listLabelsForRepo.endpoint.merge({
+        owner: this.owner,
+        repo: this.repo,
+      })
+    );
+
+    return labels.map(label => new Label(label));
+  }
+
+  /**
+   * @param {Object} operations
+   * @returns {Promise}
+   */
+  update(operations) {
+    if (this.dryRun) {
```

Comment:
Why are we using the dry run here? In the [syncer](https://github.com/rdohms/gh-labeller/pull/1/files#diff-e02504d72320c34de73a781c47a29e89R18) if we specified dry run, we return immediately, and we don't execute these functions.